v5_MIGRATION_GUIDE.md

```diff
@@ -150,3 +152,71 @@ class MyActivity : FragmentActivity() {
 }
 ```
 
+## PayPal
+
+The PayPal integration has been updated to allow for more flexibility with browser switching and
+result handling.
+
+`PayPalLauncher` has been added to handle the browser switching portion of the PayPal flow for
+payment authorization via PayPal login. This class is used to handle browser switch 
+results upon returning to your app from the web flow, therefore must be instantiated in or 
+before the `OnResume` method of your Activity or Fragment.
+
+`BraintreeClient` and `PayPalClient` no longer require references to Fragment or Activity and
+do not need to be instantiated in `OnCreate`.
+
+```diff
+class MyActivity : FragmentActivity() {
```

Comment:
Do we need to show calling `handleReturnToAppFromBrowser` in this code sample?

Reply:
Ah yes this was renamed from `deliverResult` - updated migration guide